src/inc/corinfo.h

```diff
@@ -276,12 +276,26 @@ enum SystemVClassificationType : unsigned __int8
     SystemVClassificationTypeMemory             = 3,
     SystemVClassificationTypeInteger            = 4,
     SystemVClassificationTypeIntegerReference   = 5,
-    SystemVClassificationTypeSSE                = 6,
+    SystemVClassificationTypeIntegerByRef       = 6,
+    SystemVClassificationTypeSSE                = 7,
     // SystemVClassificationTypeSSEUp           = Unused, // Not supported by the CLR.
     // SystemVClassificationTypeX87             = Unused, // Not supported by the CLR.
     // SystemVClassificationTypeX87Up           = Unused, // Not supported by the CLR.
     // SystemVClassificationTypeComplexX87      = Unused, // Not supported by the CLR.
-    SystemVClassificationTypeMAX = 7,
+
+    // Internal flags - never returned outside of the classification implementation.
+
+    // This value represents a very special type with two eightbytes. 
```

Comment:
"eightbyte" is a term used in the classification specification. The term refers to the classification meaning of the word.
Clarified that only the first eightbyte type is changed.